README.md

```diff
@@ -200,6 +200,14 @@ Supervisor.init(
 )
 ```
 
+**using [`igniter`](https://hex.pm/packages/igniter)**
```

Comment:
You can use tabsets for this in ex_doc which looks pretty nice. https://hexdocs.pm/ex_doc/readme.html#tabsets

Reply:
https://hexdocs.pm/ash/get-started.html#create-a-new-project

Reply:
Ooh, didn't know about those. I'll add them, but one level up. (Erlang vs. Elixir)